test/cmd.js

```diff
@@ -335,35 +381,6 @@ describe('express(1)', function () {
       .expect(404, /<h1>Not Found<\/h1>/, done);
     });
   });
-
-  describe('--help', function () {
```

Comment:
Hmm, maybe we should keep both.. Now using async to handle both -h and --help without duplicating 30 lines of code.